src/options/options.html

```diff
@@ -188,6 +188,13 @@ <h1 data-i18n="__MSG_titleAddonBehavior__">Addon behavior</h1>
 							<span data-i18n="__MSG_optionDebugModeDescr__" class="helper-text">This is only useful to get more detailed logging output in the console for reporting bugs, etc.</span>
 						</div>
 					</li>
+
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="popupShowContextMenu" name="popupShowContextMenu">
+							<label data-i18n="__MSG_optionPopupShowContextMenu__" for="popupIconShowContextMenu">Show context menu item for selected text</label>
```

Comment:
All `__MSG` things need localisation. (At least the English one if you speak no other language.) See [this answer again](https://github.com/rugk/offline-qr-code/issues/157#issuecomment-508475513) and [this doc](https://github.com/TinyWebEx/Localizer).